tools/gulp/tasks/publish.ts

```diff
@@ -42,19 +42,26 @@ task(':publish:whoami', execTask('npm', ['whoami'], {
 task(':publish:logout', execTask('npm', ['logout']));
 
 task(':publish', async () => {
-  const label = argv['tag'];
+  const tag = argv['tag'];
   const version = buildConfig.projectVersion;
   const currentDir = process.cwd();
 
   console.log('');
-  if (!label) {
-    console.log(grey('> You can use a label with --tag=labelName.\n'));
-    console.log(green(`Publishing version "${version}" using the latest tag...`));
+  if (!tag) {
+    console.log(grey('> You can specify the tag by passing --tag=labelName.\n'));
+    console.log(green(`Publishing version "${version}" within the latest tag...`));
   } else {
-    console.log(yellow(`Publishing version "${version}" using the ${label} tag...`));
+    console.log(yellow(`Publishing version "${version}" within the ${tag} tag...`));
   }
   console.log('');
 
+
+  if (version.includes('beta') && (!tag || tag === 'latest')) {
+    console.log(red('Publishing a beta version to the "latest" tag is not allowed. Exiting..'));
```

Comment:
I'd omit the "Exiting.." at the end.

Also, empty `console.log()`?

Reply:
Done. Kept the `console.log()` at the end, because it makes the error message way more readable inside of all the gulp messages.